### PR TITLE
Adding main function

### PR DESCRIPTION
**Pull request by moose-x86**
This change is [Reviewable](https://reviewable.io/reviews/nadzwyczajnagruparobocza/bomberman/77)

### CODE REVIEW

File: src/game/main.cpp [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    while (window.isOpen())
+    {
+        sf::Event event;
+        while (window.pollEvent(event))
+        {
+            if (event.type == sf::Event::Closed)
+                window.close();
+        }
+
+        std::chrono::milliseconds ms{clock.restart().asMilliseconds()};
```
Comment (kn65op) [inline src/game/main.cpp]: Please add const.

**Review by kn65op — APPROVED** [review]

**Comment by kn65op** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[src/game/main.cpp, line 21 at r3](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qZa28xTfUKpH_-ZeZ:-L4qZa28xTfUKpH_-Ze_:b-7iykvh) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/acc7c01070d29a215739f2c9d27fec473fe7f21a/src/game/main.cpp#L21)):*
> ```cpp
>         {
>             if (event.type == sf::Event::Closed)
>                 window.close();
> ```

test

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by Abergard** [comment]
Review status: 0 of 2 files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---

*[src/game/main.cpp, line 9 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qalxwQj8fva8nTPhQ:-L4qalxxEfRM7CseWT7f:bpezrjd) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/ac6a30b89cdaf5fff061d006327e6390aea37724/src/game/main.cpp#L9)):*
> ```cpp
> int main()
> {
>     const std::pair<int, int> window_size{800, 600};
> ```

Why not math::Size2u?

---

*[src/game/main.cpp, line 24 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qHgOn0HuZz21L41pu-r1-24:-L4qbSxYio54MUu5zzsj:b7ej898) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/ac6a30b89cdaf5fff061d006327e6390aea37724/src/game/main.cpp#L24)):*
<details><summary><i>Previously, kn65op (Tomasz Drzewiecki) wrote…</i></summary><blockquote>

Please add const.
</blockquote></details>

Ok. Its fine, however what about move it into ElapsedTime?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by Abergard** [comment]
Reviewed 1 of 2 files at r3, 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 2 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by Abergard** [comment]
Reviewed 1 of 1 files at r5.
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by moose-x86** [comment]
Review status: all files reviewed at latest revision, 2 unresolved discussions.

---

*[src/game/main.cpp, line 9 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qalxwQj8fva8nTPhQ:-L6VVLPbaEcQ9wIb8w0H:b-896fix) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/ac6a30b89cdaf5fff061d006327e6390aea37724/src/game/main.cpp#L9)):*
<details><summary><i>Previously, Abergard (Lukass) wrote…</i></summary><blockquote>

Why not math::Size2u?
</blockquote></details>

Done.

---

*[src/game/main.cpp, line 24 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qHgOn0HuZz21L41pu-r1-24:-L6VVM5oUR1OmKucyUx6:b-896fix) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/ac6a30b89cdaf5fff061d006327e6390aea37724/src/game/main.cpp#L24)):*
<details><summary><i>Previously, Abergard (Lukass) wrote…</i></summary><blockquote>

Ok. Its fine, however what about move it into ElapsedTime?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by Abergard** [comment]
Reviewed 1 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Comment by Abergard** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[src/game/main.cpp, line 24 at r1](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77#-L4qHgOn0HuZz21L41pu-r1-24:-L6VVdIHoXHMSXMwmtuF:b2nhll9) ([raw file](https://github.com/nadzwyczajnagruparobocza/bomberman/blob/ac6a30b89cdaf5fff061d006327e6390aea37724/src/game/main.cpp#L24)):*
<details><summary><i>Previously, moose-x86 (Przemek) wrote…</i></summary><blockquote>

Done.
</blockquote></details>

Ok.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/nadzwyczajnagruparobocza/bomberman/77)*

**Pull request merged**